SoftLayer/CLI/event_log/get.py

```diff
@@ -1,52 +1,54 @@
 """Get Event Logs."""
 # :license: MIT, see LICENSE for more details.
 
-import click
 import json
+import click
 
 import SoftLayer
 from SoftLayer.CLI import environment
-from SoftLayer.CLI import exceptions
 from SoftLayer.CLI import formatting
 
 COLUMNS = ['event', 'label', 'date', 'metadata']
 
+
 @click.command()
 @click.option('--obj_id', '-i',
               help="The id of the object we want to get event logs for")
 @click.option('--obj_type', '-t',
               help="The type of the object we want to get event logs for")
 @environment.pass_env
-
 def cli(env, obj_id, obj_type):
     """Get Event Logs"""
-    mgr = SoftLayer.NetworkManager(env.client)
+    mgr = SoftLayer.EventLogManager(env.client)
 
-    filter = _build_filter(obj_id, obj_type)
+    request_filter = _build_filter(obj_id, obj_type)
 
-    logs = mgr.get_event_logs(filter)
+    logs = mgr.get_event_logs(request_filter)
 
     table = formatting.Table(COLUMNS)
     table.align['metadata'] = "l"
 
     for log in logs:
-        metadata = json.loads(log['metaData'])
+        try:
+            metadata = json.dumps(json.loads(log['metaData']), indent=4, sort_keys=True)
+        except ValueError:
+            metadata = log['metaData']
 
-        table.add_row([log['eventName'], log['label'], log['eventCreateDate'], json.dumps(metadata, indent=4, sort_keys=True)])
+        table.add_row([log['eventName'], log['label'], log['eventCreateDate'], metadata])
 
     env.fout(table)
 
 
 def _build_filter(obj_id, obj_type):
     if not obj_id and not obj_type:
         return None
-    
-    filter = {}
+
+    request_filter = {}
 
     if obj_id:
-        filter['objectId'] = {'operation': obj_id}
+        request_filter['objectId'] = {'operation': obj_id}
 
     if obj_type:
-        filter['objectName'] = {'operation': obj_type}
+        request_filter['objectName'] = {'operation': obj_type}
 
-    return filter
+    return request_filter
```

Comment:
In the GUI, you can also filter by Action.

Is it possible to also filter by request id? I suppose since this CLI is not specific to security groups, that would mean a generic metadata filter is needed.

The GUI also has DateFrom and DateTo filters. That would be another way to limit the output.

Reply:
I'd have to see how those filters are built for calls to Event_Log.getAllObjects